modules/gcp-gke/cluster.tf

```diff
@@ -1,36 +1,14 @@
-terraform {
-  required_version = ">= 1.2.9"
-
-  required_providers {
-    google = {
-      version               = ">= 4.0"
-      configuration_aliases = [google.compute, google.vpc]
-    }
-
-    google-beta = {
-      version               = ">= 4.0"
-      source                = "hashicorp/google-beta"
-      configuration_aliases = [google-beta.compute-beta]
-    }
-
-    random = {
-      version = "~> 3.0"
-    }
-  }
-}
```

Comment:
We still need to define the module's provider dependencies and minimum TF version, just not any `configuration_aliases`.